example/index.ts

```diff
@@ -24,24 +25,40 @@ function runTwice() {
 
 runTwice();
 
+ */
+
 
 function runWithoutModel() {
-    elmStaticHtml(process.cwd(), "MyModule.otherView", {})
+    elmStaticHtml.default(process.cwd(), "MyModule.otherView", {})
     .then((generatedHtml) => {
         fs.writeFileSync("output4.html", generatedHtml);
     });
 }
 
 runWithoutModel();
 
-
 function runLazyView() {
-    elmStaticHtml(process.cwd(), "MyModule.lazyView", firstRunOptions)
+    elmStaticHtml.default(process.cwd(), "MyModule.lazyView", firstRunOptions)
     .then((generatedHtml) => {
         fs.writeFileSync("output5.html", generatedHtml);
     }).catch((err) =>{
         console.log(err);
     });
 } 
 
-runLazyView();
\ No newline at end of file
+runLazyView();
+
+function runMultiple() {
+    const configs =
+        [ { viewFunction: "MyModule.view", model, decoder: "MyModule.decodeModel", output: "multiple1.html" }
+        , { viewFunction: "MyModule.lazyView", model, decoder: "MyModule.decodeModel", output: "multiple2.html" }
+        ];
+
+    elmStaticHtml.grouped(process.cwd(), "MyModule", configs)
+        .then((generatedHtmls) => {
+            generatedHtmls
+                .map((generatedHtml, i) => fs.writeFileSync(configs[i].output, generatedHtml));
```

Comment:
Relying on order kinda sucks: why not make make it so that the returned stuff is a list like this:

```
List { generatedHtml : String,  fileOutputName: String }
```
?